### ISSUE TITLE
Capped option not set on new collection

### ISSUE THREAD

**Issue by jwillmer** [serilog/serilog-sinks-mongodb#68]
I'm not able to get a mongo collection with the capped option using the config example from the README. 

Below is my configuration:

```
...
        "WriteTo": [
            {
                "Name": "MongoDB",
                "Args": {
                    "databaseUrl": "mongodb://localhost:27017/site-monitor",
                    "collectionName": "monitoring.logs",
                    "restrictedToMinimumLevel": "Warning",
                    "cappedMaxSizeMb": "50",
                    "cappedMaxDocuments": "1000"
                }
            }
...
```

Do I miss something or is there a way to debug?

**Comment by jwillmer** [comment]
@Jaben can U comment on it?

**Comment by Jaben** [comment]
> @Jaben can U comment on it?

It's fixed in v5.1.2 (latest).

**Comment by wuzhenda** [comment]
the max value of the two variable?
```        
"cappedMaxSizeMb"  
"cappedMaxDocuments" 
```
